pkg/limitador/k8s_objects.go

```diff
@@ -196,3 +198,17 @@ func ownerRefToLimitador(limitador *limitadorv1alpha1.Limitador) metav1.OwnerRef
 		UID:        limitador.UID,
 	}
 }
+
+func storageType(storage *limitadorv1alpha1.Storage) string {
+	if storage != nil {
+		return string(storage.Type)
+	}
+	return string(limitadorv1alpha1.StorageTypeInMemory)
+}
+
+func storageConfig(storage *limitadorv1alpha1.Storage, storageConfigSecret *v1.Secret) string {
+	if storage != nil && storage.Type != limitadorv1alpha1.StorageTypeInMemory {
+		return string(storageConfigSecret.Data["URL"])
+	}
+	return ""
```

Comment:
Limitador's CLI does not have any issue with an empty param, does it?

If the CLI does not have any issue with empty params, it is correct. For the future, I would have one method per limitador deployment mode that returns `[]string` and the result is append to the basic command line params `[]string{"limitador-server", "/path/limits.file"}`

Reply:
You're right, it's much better

Reply:
It would have an issue with the empty arg _I think_: 
```commandline
$ /limitador-server limitador-server/examples/limits.yaml memory ""
error: Found argument '' which wasn't expected, or isn't valid in this context

USAGE:
    limitador-server <LIMITS_FILE> memory

For more information try --help
```

Which is correct… this isn't valid input an empty string _is a input argument_ and we validate all inputs to be correct. I think that's the behavior you'd observe passing that additional empty string… but I could be wrong, th go stdlib might be doing some magic… tho that magic would be surprising to me!

Reply:
As a corollary this won't work either:

```commandline
$ ./limitador-server limitador-server/examples/limits.yaml ""
error: Found argument '' which wasn't expected, or isn't valid in this context

USAGE:
    limitador-server [OPTIONS] <LIMITS_FILE> [STORAGE]

For more information try --help
```

As this isn't one of the supported `STORAGES`